### PR TITLE
Feature/add assignee information

### PR DESCRIPTION
**Pull request by nsmith-amplience**
This PR adds assignee information into the SDK to allow the Zapier integration to handle assignee webhook events.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,7 +42,8 @@
     "all": "run-s reset build test doc:html",
     "prepare-patch-release": "run-s all version:patch doc:publish",
     "prepare-minor-release": "run-s all version:minor doc:publish",
-    "prepare-major-release": "run-s all version:major doc:publish"
+    "prepare-major-release": "run-s all version:major doc:publish",
+    "prepare": "npm run build"
```
Comment (easen-amp) [inline package.json:46]: maybe this should be `npm run all`, so it can run the tests and docs tasks

File: src/lib/model/ContentItem.ts [hunk 2]
```diff
@@ -84,6 +84,16 @@ export class ContentItem extends HalResource {
...
 
+  /**
+   * List of user Id' who are assigned to the content item
+   */
+  public assignees?: string[];
+
+  /**
+   * Timestamp representing when the assignees list was last updated in ISO 8601 format
+   */
+  public assignedDate?: string;
```
Comment (easen-amp) [inline src/lib/model/ContentItem.ts:95]: Can you update the tests, to ensure these fields are accessible.

File: package.json (hunk 1 above)
Comment (nsmith-amplience) [reply package.json:46]: Done 👍

File: src/lib/model/ContentItem.ts (hunk 2 above)
Comment (nsmith-amplience) [reply src/lib/model/ContentItem.ts:95]: Added a new test showing they're available.

File: package.json (hunk 1 above)
Comment (nsmith-amplience) [reply package.json]: Looks like `npm run all` fails on Travis but works locally.

**Pull request merged**